PowerUp/app/src/main/java/powerup/systers/com/utils/CSVReader.java

```diff
@@ -0,0 +1,112 @@
+package powerup.systers.com.utils;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import powerup.systers.com.data.entities.Accessory;
+import powerup.systers.com.data.entities.Answer;
+import powerup.systers.com.data.entities.Clothes;
+import powerup.systers.com.data.entities.Hair;
+import powerup.systers.com.data.entities.Question;
+import powerup.systers.com.data.entities.Scenario;
+
+public class CSVReader {
```

Comment:
I wanted to use generic function initially but I faced problem while adding values to list. 
Number of field in different entity class in different & so as their datatypes. Can you check line 35, 49, 64, 78 once & suggest me any approach if it could be handled with Generic method?  So, I opted for different function for different required list instead.

Reply:
I can see what you are saying, there are multiple ways to fix this,  one could be:
pass input 2 input args, fileName and type, where type can determine what type of object needs to be creating for adding to List. something like this:
public List<T> getEntityListFromFile(String FileName, String type)
{
// to buffer read IO operation
// read each line
// create an object depending upon type. e..g 
// if type == "Question", then create T listObject = new<T>((Integer.valueOf(row[0]), Integer.valueOf(row[1]), row[2]); and add to List
// if type == "Answers", then create T, Integer.valueOf(row[0]), Integer.valueOf(row[1]),
                  row[2], Integer.valueOf(row[3]), Integer.valueOf(row[4]))

// and so on
}
Now, create Object you can have separate small method, to create object for each type, just to make code more modular and clean. Let me know, if you have any other suggestion.